pom.xml

```diff
@@ -114,7 +114,7 @@
     <commons-configuration.version>1.10</commons-configuration.version>
     <commons-compress.version>1.15</commons-compress.version>
     <commons-lang.version>2.6</commons-lang.version>
-    <commons-lang3.version>3.3.2</commons-lang3.version>
+    <commons-lang3.version>3.6</commons-lang3.version>
```

Comment:
It is worth to add in the description of the pr a line about this library upgrade, so that it will be stated in git commit log